src/client/voice/VoiceConnection.js

```diff
@@ -125,6 +125,20 @@ class VoiceConnection extends EventEmitter {
      * @type {VoiceReceiver}
      */
     this.receiver = new VoiceReceiver(this);
+
+    /**
+     * The state of self_deaf for the connection
+     * @type {boolean}
+     * @private
+     */
+    this.selfDeaf = false;
+
+    /**
+     * The state of self_mute for the connection
+     * @type {boolean}
+     * @private
+     */
+    this.selfMute = false;
```

Comment:
These properties should not exist on the VoiceConnection, they should just be getters, i.e. they should use `connection.voice.{selfDeaf,selfMute}`

Reply:
I haven't had time to work on this due to windows corrupting all my files and then grub deciding to not work. that is left over from the start of this pr before I started to do the changes discussed in dms